src/app/layout/Layout.tsx

```diff
@@ -14,14 +14,15 @@ export function Layout({ children }: { children: React.ReactNode }) {
     setSidebarOpen(!isSidebarOpen);
   };
 
+  const isRestrictPath =
+    pathname === Routes.Playground || pathname === Routes.Login;
+
   return (
     <div className="flex">
-      {(pathname !== Routes.Login || isSidebarOpen) && (
-        <Sidebar isSidebarOpen={isSidebarOpen} toggleSidebar={toggleSidebar} />
-      )}
+      <Sidebar isSidebarOpen={isSidebarOpen} toggleSidebar={toggleSidebar} />
       <div
         className={cx(
-          `flex flex-col w-full h-screen ${pathname !== Routes.Login && "lg:ml-48"}`
+          `flex flex-col w-full h-screen ${isRestrictPath ? "" : "lg:ml-48"}`
```

Comment:
@muhammad-bilal-ali-khokhar we should use here  `&` operation instead of `ternary` operator.

Reply:
That would cause this to return undefined @nadirhussain786